### DISCUSSION TITLE
Learndash with bootscore

### DISCUSSION THREAD

**Discussion by mhostn3** [bootscore/bootscore#298] [Q&A] [answered]
First of all, thank you for your great job building this theme.

I am using Learndash and your theme together.

But when I generate the course on Learndash it uses the page template with the sidebar.
I would like to use by default the page layout without the sidebar.
I have contacted guys from Learndash and they say it is a theme issue.

Any idea?

Thanks,
Martin

**Comment by crftwrk** [comment, marked as answer]
Thank you!

At first glance it looks that Learndash uses default `page.php` and needs like WooCommerce an own page template. If you cannot select page template in the backend, try this:

- Copy from main-theme `/page-templates/page-sidebar-none.php` to your child's root folder
- Rename `page-sidebar-none.php` to `page.php`

This is now the default template for all pages. Let me know if that works for Learndash sites, then we can investigate further.

BTW, a link to your project is awesome helpful.

**Reply by mhostn3** [reply]
Hello, thank you very much for your answer.
I have tried, but it is not working. I think Learndash takes post layout as the template. 
How should I proceed in this case?

Thanks,
Martin

**Reply by crftwrk** [reply]
Dealing with post templates is exactly the same as with page templates https://bootscore.me/documentation/page-post-templates/.

- Copy from main-theme `/single-templates/single-sidebar-none.php` to your child's root folder
- Rename `single-sidebar-none.php` to `single.php`

You can quickly check which type (page/post) LearnDash uses for courses. If there is a breadcrumb then it's a post, if there is no breadcrumb then it's a page.

**Reply by mhostn3** [reply]
Perfect! It works now! Thank you again!

**Reply by crftwrk** [reply]
You're welcome 👍
